feat(engine): 2349: explain "This expression is not callable." error (#45)
mattpocock/ts-error-translator: packages/engine/errors/2349.md

---
original: 'This expression is not callable.'

> excerpt: "I can't call this expression because it's not a callable object like a function."

Reviewer: ```suggestion
excerpt: "I can't call this expression because I can't call it like a function."
```

---

> You've called a not callable object as if it's like a function. For instance:

Reviewer: ```suggestion
You're trying to call something that doesn't appear to be a function. A version of this error might be:
```
